src/components/Gallery.vue

```diff
@@ -16,56 +16,87 @@ function convertRemToPixels(rem) {
 }
 
 const props = defineProps({
+  /**
+   * The array of items (data) to render the gallery.
+   */
   items: {
     type: Array,
     default: () => {
       return []
     },
   },
+  /**
+   * The maximum width we are allowed minus two buttons for next and previous divided by the width of a card.
+   */
   maxWidth: {
     type: Number,
     default: 3,
   },
+  /**
+   * The width of the card in rem unit.
+   */
   cardWidth: {
     type: Number,
     default: 13.8,
   },
+  /**
+   * The option to show indicator bar at the bottom.
+   */
   showIndicatorBar: {
     type: Boolean,
     default: true,
   },
+  /**
+   * The option to highlight the active item.
+   */
   highlightActive: {
     type: Boolean,
     default: true,
   },
+  /**
+   * The option to show card details.
+   */
   showCardDetails: {
     type: Boolean,
     default: true,
   },
+  /**
+   * The CSS to define card's style.
+   */
   bodyStyle: {
     type: Object,
     default: () => {
       return { padding: '20px', background: '#ffffff' }
     },
   },
+  /**
+   * The space between the card and indicator bar.
+   */
   bottomSpacer: {
     type: Object,
     default: () => {
       return { minHeight: '4rem' }
     },
   },
+  /**
+   * The CSS for image container in card.
+   */
   imageContainerStyle: {
     type: Object,
     default: () => {
       return {}
     },
   },
+  /**
+   * The CSS for image style in card.
+   */
   imageStyle: {
     type: Object,
     default: () => {
       return {}
     },
   },
+  // TODO: not in use
```

Comment:
I think this prop is causing some strange results in the documentation generation:

Reply:
It is fixed in this commit https://github.com/ABI-Software/mapcore-gallery/pull/9/commits/824fbc24fda8a596337b60b17bc96ca5df0b4728